backend/.env.development

```diff
@@ -67,11 +67,27 @@ LANGCHAIN_API_KEY=your_langsmith_api_key_here
 # To create a LangSmith project, visit LangSmith: https://www.langchain.com/langsmith
 LANGCHAIN_PROJECT=your_langsmith_project_name_here
 
-
 # FILE_UPLOAD: Whether to enable file upload to storage
-# Set to true if file upload is required.
+# Available options: false, s3, minio
 # If set to false, AWS_S3_BUCKET_NAME is not required.
-FILE_UPLOAD=false
-# AWS_S3_BUCKET_NAME: S3 Bucket name 
-# This is the name of the S3 Bucket
-AWS_S3_BUCKET_NAME="your_s3_bucket_name"
\ No newline at end of file
+# Set to true if file upload is required.
+FILE_UPLOAD=minio
+# Storage configuration for either AWS S3 or MinIO
+# This is the name of the S3 Bucket or MinIO Bucket
+BUCKET_NAME="default-storage"
+# MINIO_ENDPOINT: The endpoint URL for the MinIO server
+# Format: http(s)://<host>:<port>
+# Example: http://localhost:9000 (For development mode)
+ MINIO_ENDPOINT="http://minio:9000"
+# Common access keys for both S3 and MinIO
+# MINIO_ACCESS_KEY: Access key for authentication
+# Warning: Use a strong, unique value in production
+# Default: minioadmin (for development only)
+ MINIO_ACCESS_KEY="minioadmin"
+# MINIO_SECRET_KEY: Secret key for authentication
+# Warning: Keep this value secret and never commit to version control
+# Default: minioadmin (for development only)
+ MINIO_SECRET_KEY="minioadmin"
```

Comment:
It seems that the name of key is different with the name in `storage.module.ts`.